src/Azure.Functions.Cli/npm/lib/install.js

```diff
@@ -41,7 +42,7 @@ if (os.platform() === 'win32') {
 }
 
 const fileName = 'Azure.Functions.Cli.' + platform + '.' + version + '.zip';
-const endpoint = 'https://functionscdn.azureedge.net/public/' + version + '/' + fileName;
+const endpoint = 'https://functionscdn.azureedge.net/public/' + consolidatedBuildId + '/' + fileName;
```

Comment:
this needs to be "4.0.consolidatedBuildId"

Reply:
I have the const above modified to this :)